src/test/test.cpp

```diff
@@ -290,6 +378,14 @@ runTest(CommandLineArgs const& args)
     {
         LOG_ERROR(DEFAULT_LOG, "Nonzero test result with --rng-seed {}", seed);
     }
+    if (gTestTxMetaMode == TestTxMetaMode::RECORD)
+    {
+        saveTestTxMeta(recordTestTxMeta);
+    }
+    else if (gTestTxMetaMode == TestTxMetaMode::CHECK)
+    {
+        checkTestTxMetaCount();
```

Comment:
That test at the end doesn't seem necessary:
* if there are tests that are not run, we can't blame it on the meta checker (and not just that, it allows us to reuse meta files from reference builds on a subset of tests)
* if we require meta to always be present, on a section by section basis we'll fail properly

Reply:
Ok. I was a little concerned that by not checking this, we'd potentially be masking cases:
  - Checking machinery broken, nothing actually checked.
  - Tests changed and stopped exercising important part of state-space.

But if you prefer I will remove it.